### PR TITLE
Feat added notes page

### PR DESCRIPTION
**Pull request by raman1999**
https://user-images.githubusercontent.com/45732494/165740243-3156d638-d9d3-403b-aef2-567fbe6cd848.mp4

### CODE REVIEW

File: src/Components/FilterModal/FilterModal.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import { useFilterContext, useUserContext } from "../../Context";
+
+export function FilterModal({ setShowFilterModal }) {
+  const {
+    userState: { labels },
+  } = useUserContext();
+  const {
+    filterState: { sortByDate, priorityFilter, labelFilter },
```
Comment (rahulcs754) [inline src/Components/FilterModal/FilterModal.jsx:8]: nice to destructor all variable

**Review by rahulcs754 — APPROVED** [review]
The code looks good and clean nice design

File: src/Components/ColorPallete/ColorPallete.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { useUserContext } from "../../Context";
+import { updateNotes } from "../../Services";
+
+export function ColorPallete({ noteDetails }) {
+  const { userDispatch } = useUserContext();
+  const colorPallete = [
```
Comment (coldpigli) [inline src/Components/ColorPallete/ColorPallete.jsx:6]: can extract this to a constants file

**Review by coldpigli — APPROVED** [review]
Everything looks good. Just one minor suggestion

File: src/Components/ColorPallete/ColorPallete.jsx (hunk 2 above)
Comment (raman1999) [reply src/Components/ColorPallete/ColorPallete.jsx:6]: Thanks for suggestion,  I will do it later

**Pull request merged**